18.1.0 build from source missing mlir/Dialect/Func/IR/FuncOps.h.inc?

I download llvm-project-18.1.0.src.tar.xz and tried to build with the following:

```
[]$ cmake                                                            \
? -DLLVM_TARGETS_TO_BUILD=X86                                        \
? -DCLANG_CONFIG_FILE_USER_DIR="${SRC}/${VER}/${TAR}/llvm.cfg"       \
? -DCMAKE_C_COMPILER=${HOST_GCC}/bin/gcc                             \
? -DCMAKE_CXX_COMPILER=${HOST_GCC}/bin/g++                           \
? -DCMAKE_BUILD_TYPE="Release"                                       \
? -DLIBOMP_ARCH="x86_64"                                             \
? -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}                                       \
? -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;cross-project-tests;libc;lld;lldb;mlir;polly;pstl" \
? -DLLVM_ENABLE_RUNTIMES="compiler-rt;libc;libcxx;libcxxabi;libunwind;openmp"    \
? -DCMAKE_CXX_LINK_FLAGS="-L${HOST_GCC}/lib64 -Wl,-rpath,${HOST_GCC}/lib64"      \
? ../llvm
  .
  .
  .
[]$ make
  .
  .
  .
[ 71%] Built target MLIRSCFDialect
[ 71%] Building CXX object tools/mlir/lib/Dialect/Affine/Analysis/CMakeFiles/obj.MLIRAffineAnalysis.dir/AffineAnalysis.cpp.o
In file included from /scr/LLVM/llvm-18.1.0/llvm-project-18.1.0.src/mlir/lib/Dialect/Affine/Analysis/AffineAnalysis.cpp:20:
/scr/LLVM/llvm-18.1.0/llvm-project-18.1.0.src/mlir/include/mlir/Dialect/Func/IR/FuncOps.h:29:10: fatal error: mlir/Dialect/Func/IR/FuncOps.h.inc: No such file or directory
   29 | #include "mlir/Dialect/Func/IR/FuncOps.h.inc"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

I'm not sure what the appropriate work around is.  Maybe just remove mlir from projects?

--- a/mlir/lib/Dialect/Affine/Analysis/CMakeLists.txt
+++ b/mlir/lib/Dialect/Affine/Analysis/CMakeLists.txt
@@ -8,6 +8,9 @@ add_mlir_dialect_library(MLIRAffineAnalysis
   ADDITIONAL_HEADER_DIRS
   ${MLIR_MAIN_INCLUDE_DIR}/mlir/Dialect/Affine
 
+  DEPENDS
+  MLIRFuncOpsIncGen
+
   LINK_LIBS PUBLIC
   MLIRAffineDialect
   MLIRAnalysis

